photon-targeting/src/main/proto/photon_types.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
```

Comment:
Protobuf styleguide uses `lower_camel_case`. When it generates language bindings it might alter it. Since you are doing camel case that is why you get `bestcameratotarget` in C++ instead of `best_camera_to_target`. It will convert snake to camel when it generates java